master/usage/encrypt-comms.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Configuring encryption and authentication
+canonical_url: 'https://docs.projectcalico.org/master/usage/encrypt-comms'
+---
+
+## etcd connections
```

Comment:
I think it might help for this title to be more explicit: "Connections from Calico components to etcd"  WDYT?

Reply:
You are probably right.